project/webapp/templates/partials/feed/new_thread.html

```diff
@@ -39,7 +39,7 @@
                 <select id="thread-category">
                     <option value="" disabled selected>Choose your category</option>
                     {{# _.each(this.options.categories, function (c) { }}
-                        <option value="{{c.id}}">{{c.name}}</option>
+                        <option value="{{c.id}}" id = "thread-category-dropdown">{{c.name}}</option>
```

Comment:
This id should be moved up to the option element at L40, or else we would get multiple elements with the same id after the for each template function runs.